lib/services/profiles-api-service.js

```diff
@@ -285,15 +286,22 @@ function profileApiServiceFactory(
                     }
                 })
                 .then(function(profile) {
-                    return [profile, taskProtocol.requestProperties(node.id)];
+                    return [
+                        profile, 
+                        taskProtocol.requestProperties(node.id), 
+                        taskProtocol.activeTaskExists(node.id)
```

Comment:
There is already a same AMQP request before calling this function (https://github.com/pengz1/on-http/blob/47bdcb17e75e8a9fcf647154ba5523985a90dcc8/lib/services/profiles-api-service.js#L117). To avoid another chance of AMQP resource consumption (it might help in @scale environment), would it be possible to leverage the result got before?